doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -237,6 +237,9 @@ Non-resilient applications are sensitive to platform impairments on Compute like
 
 <p align="center"><b>Table 3-1:</b> Categories of applications, requirements for scheduling pods and Kubernetes features</p>
 
+Kubernetes clusters using above enhancements can implement worker nodes with "bare metal" servers (running Container Runtime in Linux host Operating System) or with virtual machines (VMs, on hypervisor).
+When running in VMs, the following requirements and configuration are needed for performance optimization: CPU Manager for vCPUs in VM. Huge pages enabled in hypervisor and mapped into VM. Hardware Topology Management with hypervisors enabled for NUMA. Node Feature Discovery and Device Plugin Framework for CPU instructions and devices exposed through hypervisor.
```

Comment:
These seem IaaS layer requirements for the CaaS-on-VM scenario? should we convert this to table format and add to chapter 4?

Reply:
> These seem IaaS layer requirements for the CaaS-on-VM scenario?
> Can we reword this in plain english and also write the details as chapter 4 specs in addition please?

@rgstori : Yes. Please see if commit [bfef4b9](https://github.com/cntt-n/CNTT/commit/bfef4b99d80468f4662913c4356fa53889f5af14) reads better. Then I'll create another PR for Ch4.